apps/docusaurus/docs/reference/indexer.md

```diff
@@ -0,0 +1,389 @@
+| --------------------------- | ------- | ---------------------------------------------------------------------------------------------------------------------------------------------------- |
+| amount                      | bigint  | The amount of the asset involved in the activity. Ex. 1000                                                                                           |
+| asset_type                  | String  | The type of the asset, described by a Move resource. Ex. "0x1::aptos_coin::AptosCoin"                                                                |
+| block_height                | bigint  | The blockchain height at which this activity occurred. Ex. 1500000                                                                                   |
+| entry_function_id_str       | String  | The identifier of the function called in this transaction. Ex. "transfer"                                                                            |
+| event_index                 | bigint  | Index of the event within the transaction. Ex. 1                                                                                                     |
+| gas_fee_payer_address       | String  | This is an Aptos account address that paid the gas fee for the transaction. Ex. "0x50bc83f01d48ab3b9c00048542332201ab9cbbea61bda5f48bf81dc506caa78a" |
+| is_frozen                   | Boolean | Indicates whether the asset is frozen. Ex. False                                                                                                     |
+| is_gas_fee                  | Boolean | Indicates whether this activity involved a gas fee. Ex. True                                                                                         |
+| is_transaction_success      | Boolean | Indicates whether the transaction was successful. Ex. True                                                                                           |
+| metadata                    | Object  | Use the [Hasura explorer](#indexer-api-reference) to see fields for `metadata` in this table.                                                        |
+| owner_address               | String  | This is an Aptos account address that owns the asset. Ex. "0x123abc456def7890abcdef1234567890abcdef1234"                                             |
+| owner_aptos_names           | Table   | References [owner_aptos_names](#current_aptos_names).                                                                                                |
+| owner_aptos_names_aggregate | Table   | References [owner_aptos_names](#current_aptos_names).                                                                                                |
+| storage_id                  | String  | Identifier for the storage used in the transaction. Ex. "0xa815a9a09105973084bfc31530e7c8f002846787c2f0521e1e34dc144ad83b89"                         |
```

Comment:
For fungible asset, this is the address of the object storing the token, basically the wallet address. For v1, there is no object, so we derived this field from the owner address and coin type, e.g. something like 0xjohn::AptosCoin.